Shaders/CinematicDOF.fx

```diff
@@ -442,7 +452,7 @@ namespace CinematicDOF
 	// returns minCoC
 	float PerformTileGatherHorizontal(sampler source, float2 texcoord)
 	{
-		float tileSize = TILE_SIZE * (float(BUFFER_WIDTH) / float(GROUND_TRUTH_SCREEN_WIDTH));
+		float tileSize = TILE_SIZE * (ReShade::ScreenSize.x / GROUND_TRUTH_SCREEN_WIDTH);
```

Comment:
You missed one here =)

Reply:
Hmm, I did a global search replace! :) Will correct it